tools/packaging/static-build/shim-v2/Dockerfile

```diff
@@ -2,10 +2,16 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-FROM ubuntu
+FROM ubuntu:20.04
```

Comment:
This is a valid change, but I'd like to ask you to split this into a different commit.  Also, please, check if the other Dockerfiles need the same change.

Reply:
Ok, I will remove this change. There is a nother pr #3109 which is used to delint dockerfile.